### ISSUE TITLE
Ephemeral storage limit check not skipped when `"kube-score/ignore": "container-resources"` annotation is set

### ISSUE THREAD

**Issue by zliebersbach** [zegl/kube-score#449]
**Which version of kube-score are you using?**

> kube-score version: 1.14.0

**What did you do?**

I expect the `"kube-score/ignore": "container-resources"` to ignore the ephemeral storage resource limit checks.

```shell
v1/Pod test-connection                     💥
    [CRITICAL] Container Ephemeral Storage Request and Limit
        · wget -> Ephemeral Storage limit is not set
            Resource limits are recommended to avoid resource DDOS. Set
            resources.limits.ephemeral-storage
```

Pod spec:

```yaml
apiVersion: v1
kind: Pod
metadata:
  name: "test-connection"
  annotations:
    "helm.sh/hook": test
    "kube-score/ignore": 'container-resources,container-image-pull-policy,pod-probes,container-security-context-user-group-id,container-security-context-readonlyrootfilesystem'
spec:
  containers:
    - name: wget
      image: busybox:1
      command: ['wget']
      args: ['http://my-api/health/live']
  restartPolicy: Never
```

**What did you expect to see?**

The ephemeral storage resource limit check is skipped.

**What did you see instead?**

The ephemeral storage resource limit check was not skipped.

**Comment by kmarteaux** [comment]
@zliebersbach 
Yes, this is a bug.  At present, the "container/resources" only applies to CPU and memory resource checks.  I will fix this. Until then, you can suppress the ephemeral storage test by adding this to the runtime flags specified: 

--ignore-test container-ephemeral-storage-request-and-limit

**Comment by zegl** [comment]
Fixed by #450
